project1.css

```diff
@@ -43,6 +43,7 @@
 
 .intro-text {
 	text-align: center;
+	padding-top: 100px;
```

Comment:
its better practice to add padding to the top of the section, not the intro text